### PR TITLE
fix(create): add logic to normalise setting e2eTestRunner to none if playwright or cypress

### PR DESCRIPTION
**Pull request by MetecanAydin**
[6413](https://dev.azure.com/amido-dev/Amido-Stacks/_workitems/edit/6413)

#### 📲 What

Ensure none is passed for e2eTestRunner if playwright or cypress selected.

#### 🤔 Why

As we have our own playwright and cypress implementations, we need to pass in as none.

#### 🛠 How

Logic added to the create script args normalise method to ensure expected behaviour.

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: packages/create/bin/dependencies.ts [hunk 1]
```diff
@@ -23,6 +23,12 @@ export function normaliseForwardedArgv(
     const updatedForwardArgv = forwardArgv;
     updatedForwardArgv['preset'] =
         forwardArgv['preset'] === 'next' ? 'apps' : forwardArgv['preset'];
+
+    // As playwright is not supported by the Nx generator for e2eTestRunner, we pass none to proceed and later install it ourselves as part of Stacks Playwright plugin.
+    updatedForwardArgv['e2eTestRunner'] =
+        updatedForwardArgv['e2eTestRunner'] === 'playwright'
+            ? 'none'
+            : updatedForwardArgv['e2eTestRunner'];
```
Comment (Danphillipz) [inline packages/create/bin/dependencies.ts:31]: We need to do this for both Playwright and Cypress, forwarding Cypress to the Nx create script would then cause errors when it comes to calling our own Cypress plugin

Comment (MetecanAydin) [reply packages/create/bin/dependencies.ts:31]: ah true. Well makes the fix simpler :)

**Review by Danphillipz — CHANGES_REQUESTED** [review]
Could we please add some e2e test coverage for the create script, I don't think we have any tests which actually use these? (create-e2e)

File: e2e/create-e2e/tests/create.spec.ts [hunk 2]
```diff
@@ -188,4 +188,40 @@ describe('create', () => {
             ),
         ).not.toThrow();
     });
+
+    it('can install with playwright set as e2eTestRunner', async () => {
+        const run = () =>
+            execSync(
+                'npx --yes @ensono-stacks/create-stacks-workspace@latest proj --business.company=Ensono --business.domain=Stacks --business.component=Nx --cloud.platform=azure --cloud.region=euw --domain.internal=nonprod.amidostacks.com --domain.external=prod.amidostacks.com --terraform.group=tf-group --terraform.storage=tf-storage --terraform.container=tf-container --vcs.type=github --preset=apps --appName=test-app --e2eTestRunner=playwright --no-nxCloud --skipGit --no-interactive --verbose',
```
Comment (Danphillipz) [inline e2e/create-e2e/tests/create.spec.ts:195]:
```suggestion
                'npx --yes @ensono-stacks/create-stacks-workspace@latest proj --business.company=Ensono --business.domain=Stacks --business.component=Nx --cloud.platform=azure --cloud.region=euw --domain.internal=nonprod.amidostacks.com --domain.external=prod.amidostacks.com --terraform.group=tf-group --terraform.storage=tf-storage --terraform.container=tf-container --vcs.type=github --preset=next --appName=test-app --e2eTestRunner=playwright --no-nxCloud --skipGit --no-interactive --verbose',
```

**Review by Danphillipz — APPROVED** [review]

**Pull request merged**